src/main.cpp

```diff
@@ -560,10 +560,6 @@ int main ( int argc, char** argv )
                              eLicenceType );
             if ( bUseGUI )
             {
-                // special case for the GUI mode: we want the licence type to be
```

Comment:
If the server GUI is started for the very first time (freh install), the CC licence check shall be enabled by default.